universum/modules/output/terminal_based_output.py

```diff
@@ -28,6 +29,10 @@ class TerminalBasedOutput(BaseOutput):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.block_level = 0
+        if locale.getpreferredencoding() == "UTF-8":
+            self.unicode_acceptable = True
```

Comment:
In my opinion the intermediate variable is not needed.
You can directly initialize self.block_end here.